--- src/manifest.xalan-interpretive	2011-08-10 12:38:23.892219017 -0400
+++ src/manifest.xalan-interpretive	2011-08-10 12:38:32.032219017 -0400
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Created-By: @java.version@ (@java.vendor@)
 Main-Class: org.apache.xalan.xslt.Process
-Class-Path: xercesImpl.jar xml-apis.jar serializer.jar
+Class-Path: xercesImpl.jar xml-apis.jar xalan-j2-serializer.jar
 
 Name: org/apache/xalan/
 Comment: Main Xalan engine implementing TrAX/JAXP
